docs/ref/csrf.txt

```diff
@@ -230,25 +231,21 @@ How it works
 
 The CSRF protection is based on the following things:
 
-#. A CSRF cookie that is based on a random secret value, which other sites
-   will not have access to.
+#. A CSRF cookie that is a random secret value, which other sites will not have
+   access to.
 
-   This cookie is set by ``CsrfViewMiddleware``. It is sent with every
-   response that has called ``django.middleware.csrf.get_token()`` (the
-   function used internally to retrieve the CSRF token), if it wasn't already
-   set on the request.
+   This cookie is set by ``CsrfViewMiddleware``. It is sent e.g. with every
```

Comment:
I'm not sure the "e.g." is needed?

Reply:
There are other cases where the cookie is sent, like if the original cookie had the wrong format and so needs to be reset, or if `rotate_token()` was called. I added e.g. so the reader doesn't get the mistaken impression that `get_token()` being called is the only thing that can cause the cookie to be sent.

Reply:
Sorry, I should have explained why I found this tricky rather than jumping to an incorrect alternative. 

What I'm trying to say is that the way it is currently phrased is tripping me up with the 'e.g.' in the middle of the sentence.

Reply:
Okay, I'll rephrase.